addons/rose/ember-cli-build.js

```diff
@@ -7,6 +7,9 @@ module.exports = function (defaults) {
     // Add options here
   });
 
+  app.import('node_modules/codemirror/lib/codemirror.css');
```

Comment:
Not blocking:
I am talking without the context and the knowledge of using/consuming this addon. Perhaps this is the only way of making it work and I am totally fine with it but I did not see on any of our addons or UI's we import `.css` using `app.import`. There is a reason for it? maybe we can explore importing those within _index.scss, [link here](https://github.com/hashicorp/boundary-ui/blob/main/addons/rose/addon/styles/rose/_index.scss)

Or copy those styles within `addons/rose/addon/styles/rose/components/code-editor/_index.scss`?

Take this comment as I am just thinking loud :sweat_smile:

Reply:
I had played around with trying to import it like that but it didn't seem to work (seems like the files are in the  root `node_modules` and not the `rose` one). 

I ended up following what cloud UI did and the [documentation](https://guides.emberjs.com/release/addons-and-dependencies/#toc_compiling-assets) made it seem like it was reasonable.

Let me know if there's any other ideas, I'm open to trying other things for this too.

Reply:
All good!! you already tried and followed best practices